l10n_fi_liikekirjuri/.gitignore

```diff
@@ -0,0 +1 @@
+.*
```

Comment:
Move gitignore to the root.
You may prefer to use the one at https://github.com/OCA/maintainer-quality-tools/tree/master/sample_files